vimeo-networking/src/main/java/com/vimeo/networking/logging/LoggingInterceptor.java

```diff
@@ -125,7 +125,7 @@ private String toPrettyFormat(String jsonString) {
                 JsonParser parser = new JsonParser();
                 JsonObject json = parser.parse(jsonString).getAsJsonObject();
 
-                Gson gson = new GsonBuilder().setPrettyPrinting().create();
+                Gson gson = VimeoNetworkUtil.getGsonBuilder().setPrettyPrinting().create();
```

Comment:
Do you think it would help to actually log the exception in the log below? Does it have any useful info?